build/Dockerfile

```diff
@@ -166,7 +166,7 @@ RUN --mount=type=secret,id=nginx-repo.crt,dst=/etc/ssl/nginx/nginx-repo.crt,mode
 	&& if [ -z "${NAP_MODULES##*waf*}" ]; then \
 	curl -fsSL https://cs.nginx.com/static/files/app-protect-8.repo > /etc/yum.repos.d/app-protect-8.repo; \
 	sed -i "0,/centos/s;;${NGINX_PLUS_VERSION}/centos;" /etc/yum.repos.d/app-protect-8.repo; \
-	dnf --nodocs install -y app-protect app-protect-attack-signatures app-protect-threat-campaigns; \
+	dnf --nodocs install -y app-protect app-protect-compiler app-protect-common app-protect-engine; \
```

Comment:
Ditto to above. Also do we definitely need theses additional packages? I don't see them mentioned in the docs https://docs.nginx.com/nginx-app-protect-waf/admin-guide/install/#rhel-ubi8-docker-deployment-example

Reply:
applied